proxy/Plugin.h

```diff
@@ -26,6 +26,15 @@
 #include <string>
 #include "tscore/List.h"
 
+enum plugin_dynamic_reload_mode {
```

Comment:
This is C++, there is no need to `typedef` or `using` an `enum`. Just name it "PluginDynamicReloadMode".

Reply:
Fixed this.